framework/Models/Model.py

```diff
@@ -377,6 +378,83 @@ def submit(self, myInput, samplerType, jobHandler, **kwargs):
     ## class and pass self in as the first parameter
     jobHandler.addJob((self, myInput, samplerType, kwargs), self.__class__.evaluateSample, prefix, metadata=metadata, modulesToImport=self.mods, uniqueHandler=uniqueHandler)
 
+  def createExportDictionaryFromFinishedJob(self,finishedJob, addJobId = False, inputParams=[]):
+    """
+      Method that is aimed to create a dictionary with the sampled and output variables that can be collected by the different
+      output objects.
+      @ In, finishedJob, InternalRunner object, instance of the run just finished
+      @ In, addJobId, bool, optional, add prefix in the exportDictionary? Default: False
+      @ In, inputParams, list, optional, list of input space parameters in the output object? Default: empty list
```

Comment:
As you are probably aware, it's dangerous to default an entry to a pass-by-reference object in Python, as that object will persist on future calls of the method.

The suggested fix is to default "inputParams" to "None", and then set it to "[]" if it is None at the top of the method.